frankenstein/notes.md

```diff
@@ -3,17 +3,12 @@
 Design Doc: https://docs.google.com/document/d/1C7yhMnb1x2sfeoe45f4mnnKConvroWhJ8KQZwIHJOuw/edit#heading=h.f7lkb8wswewc
 
     GOOS=linux DOCKER_IMAGE_TAG=latest make build docker docker-frank
-    ./frankenstein/run.sh
```

Comment:
They should use weave cloud...

But seriously, this old method had bit rotted and was misleading.  Id rather have no instructions than instructions that don't work.